.eslintrc.json

```diff
@@ -21,7 +24,10 @@
                     }
                   ]
                 }
-              ]
+              ],
```

Comment:
Can we move these configuration to package level? Or for the particular lines. Don't think we want to turn the rule off completely?

Reply:
Sorry these rules were leftovers from the 2.6.1 upgrade, I'll remove these for now